README.md

```diff
@@ -2,7 +2,7 @@
 <!-- PROJECT DESCRIPTION -->
 
 # 📖  <a name="about-project">blog app</a>
-The project test validations and models of the Blog application with the rspec.
+The project implements the two differents controllers and the correponding actions in the view.
 
 **blog app** is a blog application
 
```

Comment:
- [ ] Kindly use Microverse Readme template without deleting the first section. It is a requirement in all the projects.